.drone.star

```diff
@@ -2590,3 +2590,16 @@ def dependsOn(earlierStages, nextStages):
                 nextStage["depends_on"].append(earlierStage["name"])
             else:
                 nextStage["depends_on"] = [earlierStage["name"]]
+
+def calculateDiffContainsUnitTestsOnly():
+    return [
+        {
+            "name": "calculate-diff",
+            "image": "owncloudci/nodejs:14",
+            "pull": "always",
+            "commands": [
+                "bash -x tests/drone/if-diff-has-unit-tests-only.sh",
+                "ls -la",
```

Comment:
This is helpful to be able to see if the file `runUnitTestsOnly` was created (or not).
We can leave it for now, so that we can debug after merging.